src/Shared/Resources/Strings.shared.resx

```diff
@@ -208,6 +208,10 @@
     <value>MSB5023: Error parsing the nested project section in solution file. A project with the GUID "{0}" is listed as being nested under project "{1}", but does not exist in the solution.</value>
     <comment>{StrBegin="MSB5023: "}UE: The solution filename is provided separately to loggers.</comment>
   </data>
+  <data name="SolutionContainsCyclicallyNestedProjectFiles" Visibility="Public">
+    <value>MSB5025: Nested project section in solution file "{0}" has a circular nesting.</value>
```

Comment:
You're using MSB5025 in #4899

Reply:
I know, but that's an easy change, so to (hopefully) avoid confusion as to why I skipped some numbers, I took the next one instead.